src/main/java/org/terasology/physicalstats/system/PhysicalStatsSystem.java

```diff
@@ -76,10 +81,11 @@ public void initialise() {
      */
     public boolean updateHealth(EntityRef e, HealthComponent h, PhysicalStatsComponent p)
     {
-        if (h.maxHealth != p.constitution * 10) {
+        int newHealth = p.constitution * constitutionMultiplier;
```

Comment:
Maybe name this newMaxHealth for clarity

Reply:
```suggestion
        int newMaxHealth = p.constitution * constitutionMultiplier;
```